pkg/asset/agent/image/ignition.go

```diff
@@ -145,14 +149,18 @@ func (a *Ignition) Generate(dependencies asset.Parents) error {
 		}
 	}
 
+	infraEnvID := strfmt.UUID(uuid.New().String())
+	log.Debug("Generated random infra-env id ", infraEnvID)
```

Comment:
```suggestion
	logrus.Debug("Generated random infra-env id ", infraEnvID)
```

Reply:
Imported as `log "github.com/sirupsen/logrus"` still want to change :thinking: ?

Reply:
my 2c: if I see `log`, I expect it to be a `*Logger`, not the package,